boot-cos/runqemu.sh

```diff
@@ -29,16 +30,15 @@ setup() {
 	LOOPDEV=`losetup -f`
 	mkdir -p $TMPDIR
 	dd if=/dev/zero of=$UDISKPATH bs=1M count=256
-	sgdisk -n 0:0:+10M -c 0:kernel $UDISKPATH
-	sgdisk -n 0:0:0 -c 0:rootfs $UDISKPATH
+	sfdisk $UDISKPATH < ./uboot.sfdisk
 
 	sudo losetup $LOOPDEV $UDISKPATH
 	sudo partprobe $LOOPDEV
-	sudo mkfs.ext4 $LOOPDEVNAME
+	sudo mkfs.ext2 $LOOPDEVNAME
 
 	mkdir -p $LOCALDIRNAME
-	sudo mount -t ext4 $LOOPDEVNAME $LOCALDIRNAME
-	sudo cp $KERNELDIR/$KERNEL $LOCALDIRNAME
+	sudo mount -t ext2 $LOOPDEVNAME $LOCALDIRNAME
+	sudo arm-none-eabi-objcopy -O binary $KERNELDIR/$KERNEL $LOCALDIRNAME/$KERNELBIN
```

Comment:
Don't think this should be here. Its a build instruction in making the binary, so it should really be in our makefile.